### PR TITLE
Merge branch 'byond510' into master

### PR DESCRIPTION
**Pull request by tigercat2000**
Merges 510 branch into master, with some conflict fixes and other minor fixes.

@TheDZD @Fox-McCloud `byond510` branch needs to be manually deleted when this is merged

### CODE REVIEW

**Comment by monster860** [comment]
So we can finally get spess parallax merged. Wow. Also, THE FASTNESS BYOND TIME today will be SS13. Awesome!

**Comment by tigercat2000** [comment]
@monster860 510 still doesn't "kill lag" and please stop acting like it does.

**Comment by QQCucumber** [comment]
Thank you so much for this.

File: code/modules/mob/living/life.dm [hunk 1]
```diff
@@ -161,7 +161,7 @@
 
 /mob/living/proc/handle_disabilities()
 	//Eyes
-	if(disabilities & BLIND || stat)	//blindness from disability or unconsciousness doesn't get better on its own
+	if(sdisabilities & BLIND || stat)	//blindness from disability or unconsciousness doesn't get better on its own
```
Comment (Fox-McCloud) [inline code/modules/mob/living/life.dm]: I really hate there's `sdisaiblities` and `disabilities` when just `disabilities` for all of them would be fine...

Comment (tigercat2000) [inline code/modules/mob/living/life.dm]: @Fox-McCloud except, it wouldn't
both are practically up to 16 bits already

**Comment by Iamgoofball** [comment]
@tigercat2000 if it isn't killing lag, you implemented the lag fixes wrong

or this is a GNU Plus Linux deal, stallman jr.?

**Comment by tigercat2000** [comment]
@Iamgoofball It does nothing to help with the majority of our shoddy systems. tick_usage is only integrated into our process scheduler.
Most of the "lag" everyone experiences is because our host is having major network problems right now.

**Pull request merged**